src/components/screens/wallet/overview/accountInfo/identity.js

```diff
@@ -1,54 +1,116 @@
-import React, { useState } from 'react';
+import React from 'react';
 
 import { truncateAddress } from '@utils/account';
+import Tooltip from '@toolbox/tooltip/tooltip';
+import CopyToClipboard from '@toolbox/copyToClipboard';
 import styles from './accountInfo.css';
 
+const Legacy = ({ legacyAddress }) => (
+  <div className={styles.legacy}>
+    <span className={`${styles.text} ${styles.label}`}>Old address</span>
+    <CopyToClipboard
+      text={legacyAddress}
+      className={styles.text}
+    >
+      {legacyAddress}
+    </CopyToClipboard>
+  </div>
+);
+
+const WithTitle = ({
```

Comment:
Can't we abstract WithTitle and WithoutTitle into one component?

Reply:
It was together first, it had too many conditions, I preferred a little bit of duplication over a soup of ternaries.